Pickers/SWActionSheet.m

```diff
@@ -87,13 +87,25 @@ - (UIWindow *)window
     }
     else
     {
-        return SWActionSheetWindow = ({
-            UIWindow *window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
-            window.windowLevel        = self.windowLevel;
-            window.backgroundColor    = [UIColor clearColor];
-            window.rootViewController = [SWActionSheetVC new];
-            window;
-        });
+        UIWindow *window = nil;
+        if (@available(iOS 13.0, *)) {
```

Comment:
I will wrap it in #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000 , since we want to support older xcode versions for a while. for discussion, please look at end of PR
#421